### PR TITLE
Major updates to wiki.

### PR DESCRIPTION
**Pull request by DevSpen**
# Updated Wiki
* Added all functions.
* New guides.
* Fixed/cleaned up some old guides.

**This is a draft pull request and will be completed soon!** ❤️

### CODE REVIEW

**Comment by Noituri** [comment]
Woah awesome

File: src/resources/bdfdCreation.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### Who Develops BDFD?
+| Name | Discord Tag | Position 
+| ------------- | ------------- | -------------
+Jakub Tomana | wiadr000#1947 (kubastick) | Owner/Developer
+Mikołaj Radkowski  | noituri#2489 | Developer/DevOp
+Bartłomiej Skoczeń | MineBartekSA#4899 | Developer/DevOp
+
+### The Company
+*[Nilpointer Software](https://nilpointer.software/)* is the parent company of *Bot Designer For Discord*. *Nilpointer Software* a start-up focused on providing fast and quality software.
```
Comment (Noituri) [inline src/resources/bdfdCreation.md:12]: It should be Nil**P**ointer Software instead of Nil**p**ointer Software

**Review by Noituri — CHANGES_REQUESTED** [review]

File: src/resources/bdfdCreation.md (hunk 1 above)
Comment (DevSpen) [reply src/resources/bdfdCreation.md]: Resolved 👍

**Comment by DevSpen** [comment]
Imk if you see any other errors, I'm finishing some guides and modifications. Then we should be good to go.

**Comment by Noituri** [comment]
Looks great so far

**Review by Noituri — APPROVED** [review]

**Pull request merged**